src/App.js

```diff
@@ -1,24 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Routes from "Routes";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
```

Comment:
your navbar will be included in every route so best to pull it out of the routes so it doesnt rerender

Reply:
@ljmerza Thanks for the suggestion, I've refactored and moved the NavBar component up one level [(ee21a46).](https://github.com/bilafish/super-store/pull/1/commits/ee21a464333f131dbe387040bb4ae909ef772474) Initially, I was thinking to handle, other app pages which may not have a NavBar.